stateresolutionv2.go

```diff
@@ -155,13 +155,6 @@ func ResolveStateConflictsV2(
 		}
 	}
 
-	// Before we can do anything with the conflicted and unconflicted sets, we
-	// first need to auth and apply the entire auth chain in order. This is so that
-	// when we come to auth any future events against the partial state, we'll have
-	// the knowledge from the auth chain to help us to make a correct decision.
-	authEvents = r.reverseTopologicalOrdering(authEvents, TopologicalOrderByAuthEvents)
-	r.authAndApplyEvents(authEvents)
```

Comment:
Why don't we do this anymore?

Reply:
It's no longer necessary, since we can pick out from the auth events in `authAndApplyEvent` if needed. I think this actually was a bug that resulted in an incorrect partial state up front.